### DISCUSSION TITLE
Bug report: UWP App Icon disappearance

### DISCUSSION THREAD

**Discussion by Shomnipotence** [valinet/ExplorerPatcher#974] [General]
![image](https://user-images.githubusercontent.com/6630660/156506601-dc2b6b51-f5f8-4f0b-967e-caebe4f5149c.png)

OS Version: Build 22563.ni-release.220219-1637
App Version: Pre-release 22000.493.42.26

**Comment by valinet** [comment]
Not EP. It could be as well a bug in Windows. EP is not tested on Insider Preview.

**Reply by Shomnipotence** [reply]
> Not EP. It could be as well a bug in Windows. EP is not tested on Insider Preview.

It may be but the App with Version 22000.469.41.16 works well

**Reply by valinet** [reply]
It may be the same as #973 which I know about and will be fixed at some point.

**Comment by seanawesome** [comment]
I've been having this issue for months
